packages/react-components/react-table/library/package.json

```diff
@@ -17,7 +17,8 @@
     "@fluentui/react-conformance": "*",
     "@fluentui/react-conformance-griffel": "*",
     "@fluentui/scripts-api-extractor": "*",
-    "@fluentui/scripts-cypress": "*"
+    "@fluentui/scripts-cypress": "*",
+    "@fluentui/react-combobox": "*"
```

Comment:
this isn't necessary as this package is explicitly imported thus part of nx dependency graph. please remove - this will also remove need of change file for react-table